data/json/recipes/recipe_ammo.json

```diff
@@ -117,11 +117,11 @@
   "reversible": false,
   "autolearn": true,
   "book_learn": [[ "glassblowing_book", 5 ]],
+  "qualities":[
+    {"id":"GRIP","level":3}
```

Comment:
Indeed, I apologize.

I had been flustrated that I was being asked to do something that I was unable to do, and I felt that #18523 and #18536 , after the bugs were fixed, were mergeable without using the linter.